tests/gtest/avif_fuzztest_properties.cc

```diff
@@ -74,22 +85,14 @@ inline auto ArbitraryProp() {
   auto fourcc = fuzztest::Arbitrary<std::array<uint8_t, 4>>();
   auto uuid = fuzztest::Arbitrary<std::array<uint8_t, 16>>();  // ignored
   auto body = fuzztest::Arbitrary<std::vector<uint8_t>>();
-  // Don't return known properties.
-  return fuzztest::Filter(
-      [](const TestProp& prop) {
-        return !avifIsKnownPropertyType(prop.fourcc.data());
-      },
-      fuzztest::StructOf<TestProp>(fourcc, uuid, body));
+  return fuzztest::StructOf<TestProp>(fourcc, uuid, body);
 }
 
 inline auto ArbitraryUUIDProp() {
   auto fourcc = fuzztest::Just(std::array<uint8_t, 4>{'u', 'u', 'i', 'd'});
   auto uuid = fuzztest::Arbitrary<std::array<uint8_t, 16>>();
   auto body = fuzztest::Arbitrary<std::vector<uint8_t>>();
-  // Don't use invalid UUIDs.
-  return fuzztest::Filter(
-      [](const TestProp& prop) { return avifIsValidUUID(prop.uuid.data()); },
```

Comment:
I guess a random 16-byte is likely to be an invalid UUID more than 90% of the time in the first 100 tries. It may make sense to replace line 87 (`auto uuid = fuzztest::Arbitrary<std::array<uint8_t, 16>>();`) with custom code that generates a valid UUID.